src/common/XmlTestExecutionVisitor.cs

```diff
@@ -40,7 +41,7 @@ XElement CreateTestResultElement(ITestResultMessage testResult, string resultTex
                     new XAttribute("name", XmlEscape(testResult.Test.DisplayName)),
                     new XAttribute("type", testResult.TestCase.TestMethod.TestClass.Class.Name),
                     new XAttribute("method", testResult.TestCase.TestMethod.Method.Name),
-                    new XAttribute("time", testResult.ExecutionTime.ToString()),
+                    new XAttribute("time", testResult.ExecutionTime.ToString(CultureInfo.InvariantCulture)),
```

Comment:
Actually, since they're not specifying a custom format here, maybe it would be better to just not call ToString() here at all... but maybe that'd look too inconsistent when compared to the other fixes.

XAttribute will format decimals using InvariantCulture.